### ISSUE TITLE
Schedule post contains a useless checkbox to make it pending

### ISSUE THREAD

**Issue by davilera** [WordPress/gutenberg#12786]
**Describe the bug**
A _scheduled_ post has a _Pending Review_ checkbox in the _Status & Visibility_ section of the _Document_, even though it's impossible to move from _Scheduled_ to _Pending_.

**To Reproduce**
1. Schedule a post
2. There's a checkbox to set it as _Pending Review_ (even though one can't save this change).

**Expected behavior**
There are two possible solutions:

* A _Scheduled_ post should not have the _Pending Review_ checkbox.
* If it has said checkbox, then enabling it should change the _Switch to Draft_ action link to _Switch to Pending_.

In my opinion, the first solution is the best, as it follows the same behavior found in _Published_ posts. That is, only _Drafts_ can become _Pending_.

**Additional context**
- WordPress 5.0.

**Comment by sarahmonster** [comment]
I don't have a great deal of familiarity with the technical implementations of the "pending review" feature, so I did [a bit of reading](https://publishpress.com/blog/pending-review-draft/) and then tested this out to see how it currently functions. Here's a preview:

![2019-05-08 00 16 59](https://user-images.githubusercontent.com/376315/57338825-ae8eec00-70e3-11e9-816a-06f1a3cdd0e0.gif)

I followed these steps:

1. Write post.
2. Set post publish date to future date.
3. Press "Schedule" button.
4. After post is saved, check the "Pending review" checkbox.
5. Save the post by pressing "schedule" button again.
6. After post is saved, the "Pending review" checkbox clears and becomes unchecked.

I'd agree with @davilera that this behaviour is counter-intuitive, but I think that solution #2 suggested might be the better direction from a user experience perspective. Is there a reason why you wouldn't want to schedule a post, but have that post only publish once it's been reviewed? An ideal flow here might work something like this:

1. Schedule post for 17 May.
2. Select "pending review" to indicate post needs review prior to publishing.
3. If post is approved prior to 17 May, post is published on 17 May as scheduled.
4. If post is approved after 17 May, post is published immedately after being approved, using the publish date as entered on the post.

In short, without any solid data on how people use this feature, we'd probably want to keep the functionality as predictable, flexible, and consistent as possible. My suspicion is that allowing users to schedule a post pending review will allow for more flexibility whilst keeping the UI more consistent, but I'd love some additional feedback from people who may have more experience with this feature than I do! 😄

**Comment by sarahmonster** [comment]
The design team discussed this during a [triage session in Slack today](https://wordpress.slack.com/archives/C02S78ZAL/p1559059886061800) (Note: You may need a [Slack account](https://make.wordpress.org/chat/) to log in.)

Consensus was that my suggested approach here seems sensible, so let's go that route and see about getting a PR up.

**Comment by shawfactor** [comment]
The expected behaviour suggested is wrong. A post with a post date in the future should be capable of being in draft, pending, or scheduled status.

Post date is just an attribute that a user with edit_posts should be able to set and then leave in draft if the post requires more work or push pending if it requires approval.

**Comment by talldan** [comment]
This UI is now completely different. You can't have a scheduled post that's set as pending:
![Screenshot 2024-07-24 at 1 14 34 PM](https://github.com/user-attachments/assets/36021b6f-71fa-46bd-945a-3b815b83a3ed)

I'm going to close this, and suggest that if there are problems with the new implementation that new more relevant issues are created.
